test/gradcheck.jl

```diff
@@ -820,6 +819,18 @@ end
     @test back′(C̄)[1] isa Diagonal
     @test diag(back′(C̄)[1]) ≈ diag(back(C̄)[1])
   end
+  @testset "cholesky - Hermitian" begin
+    rng, N = MersenneTwister(123456), 3
+    A = randn(rng, N, N) + im * randn(rng, N, N)
```

Comment:
I think you can just use
```suggestion
    A = randn(rng, Complex{Float64}, N, N)
```

Reply:
thank you, much better :+1:

Reply:
I thought there should have been something like this:)